src/v/cluster/cloud_metadata/cluster_manifest.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+#pragma once
+
+#include "cloud_storage/base_manifest.h"
+#include "cluster/cloud_metadata/types.h"
+#include "model/fundamental.h"
+#include "seastarx.h"
+#include "serde/envelope.h"
+#include "utils/named_type.h"
+
+#include <seastar/core/lowres_clock.hh>
+
+#include <rapidjson/document.h>
+
+namespace cluster::cloud_metadata {
+
+// Manifest containing information about cluster metadata.
+//
+// NOTE: while it's not a requirement to be serialized with serde, we'll use
+// serde-style versioning to simplify reasoning about compatibility.
+struct cluster_metadata_manifest
+  : public serde::envelope<
+      cluster_metadata_manifest,
+      serde::version<0>,
+      serde::compat_version<0>>
+  , public cloud_storage::base_manifest {
+    using duration = ss::lowres_clock::time_point::duration;
```

Comment:
this clock is monotonic, since this timestamp goes outside of the redpanda process the system clock should be used

Reply:
Done